main/src/com/google/refine/importers/SeparatorBasedImporter.java

```diff
@@ -98,9 +101,15 @@ public void parseOneFile(
         boolean processQuotes = JSONUtilities.getBoolean(options, "processQuotes", true);
         boolean strictQuotes = JSONUtilities.getBoolean(options, "strictQuotes", false);
         
+        Character quote = CSVParser.DEFAULT_QUOTE_CHARACTER;
+        String quoteCharacter = JSONUtilities.getString(options, "quoteCharacter", null);
+        if (!StringUtils.isBlank(quoteCharacter)) {
+            quote = quoteCharacter.charAt(0);
```

Comment:
should this be handled at front end to only allow 1 char(or possible restriction on what char cannot be used)? If use input [blank]'[blank] then it will cause problem.

Reply:
The front end already enforces that the maximum length of this string is 1, but sure, we can make that more robust for people who do not use the web frontend.